.devcontainer/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:latest
+FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:v0.0.1
```

Comment:
Why having this small version number for the base image?
I'd suggest using at least v0.1.0 or even v1.0.0 ...

Reply:
Good question. We have just never released it before, and just decided to have the very first version